src/main.h

```diff
@@ -359,7 +359,7 @@ bool CheckInputs(const CTransaction& tx, CValidationState &state, const CCoinsVi
                  unsigned int flags, bool cacheStore, std::vector<CScriptCheck> *pvChecks = NULL);
 
 /** Apply the effects of this transaction on the UTXO set represented by view */
-void UpdateCoins(const CTransaction& tx, CValidationState &state, CCoinsViewCache &inputs, int nHeight);
+void UpdateCoins(const CTransaction& tx, CCoinsViewCache &inputs, int nHeight);
```

Comment:
nit: If you do trivial fixes, please also do all other trivial fixes in the same line:

```diff
diff --git a/src/main.cpp b/src/main.cpp
index c5d2e23..7d5beab 100644
--- a/src/main.cpp
+++ b/src/main.cpp
@@ -1747 +1747 @@ void static InvalidBlockFound(CBlockIndex *pindex, const CValidationState &state
-void UpdateCoins(const CTransaction& tx, CCoinsViewCache &inputs, CTxUndo &txundo, int nHeight)
+void UpdateCoins(const CTransaction& tx, CCoinsViewCache& inputs, CTxUndo& txundo, int nHeight)
@@ -1773 +1773 @@ void UpdateCoins(const CTransaction& tx, CCoinsViewCache &inputs, CTxUndo &txund
-void UpdateCoins(const CTransaction& tx, CCoinsViewCache &inputs, int nHeight)
+void UpdateCoins(const CTransaction& tx, CCoinsViewCache& inputs, int nHeight)
diff --git a/src/main.h b/src/main.h
index 716378e..792747a 100644
--- a/src/main.h
+++ b/src/main.h
@@ -362 +362 @@ bool CheckInputs(const CTransaction& tx, CValidationState &state, const CCoinsVi
-void UpdateCoins(const CTransaction& tx, CCoinsViewCache &inputs, int nHeight);
+void UpdateCoins(const CTransaction& tx, CCoinsViewCache& inputs, int nHeight);